### PR TITLE
Fix test bug and adjust CI

### PR DESCRIPTION
**Pull request by paulmey**
This bug was not caught by CI, because CI was not building (or executing) tests in ./management.

CI now builds everything, tests storage and management. management integration tests are set up to skip if no subscription is provided.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,5 +3,6 @@ sudo: false
 language: go
 go: tip
 script:
+    - go build -v ./...
     - go test -v ./storage/...
-    - go build -v ./management/...
+    - go test -v ./management/...
```
Comment (ahmetalpbalkan) [inline .travis.yml]: doesn't this now mean `go test -test.v ./...` ?

**Comment by ahmetalpbalkan** [comment]
actually it's better to skip integration tests with `-test.short` flag but not providing subscription apparently gives the same behavior here. 

LGTM.

File: .travis.yml (hunk 1 above)
Comment (paulmey) [inline .travis.yml]: the ./core dir also has some tests... not all of them succeed... didn't look why they fail.

Comment (ahmetalpbalkan) [inline .travis.yml]: ah ok

**Pull request merged**